lalrpop/src/session.rs

```diff
@@ -58,6 +58,8 @@ pub struct Session {
     /// this value if we so choose.
     pub max_errors: usize,
 
+    pub macro_recursion_limit: u16,
```

Comment:
This commit looks great! Thanks. A minor nit is that we can say something like `/// Macro limit ensures that compilation terminates in a finite number of steps` since most of the other fields have public documentation.

Reply:
Good catch, thanks.  Pushed.